Makefile

```diff
@@ -72,3 +72,7 @@ docs-serve: ## Serve the site locally with hot-reloading
 .PHONY: docs-build
 docs-build: ## Build the documentation site
 	cd docs && zola build
+
+.PHONY: linter-markdown
+linter-markdown: ## Lint markdown files
+	npx markdownlint-cli2  "**/*.md" "#docs"
```

Comment:
Do we need to add `npm` & `npx` as prerequisites somewhere?

Reply:
I am thinking this should be a github action that happens automatically. I dont think we want to install more tools. What do others think?

```
jobs:
  lint-changelog:
    name: Lint changelog file
    runs-on: ubuntu-latest
    steps:
    - name: Check out code
      uses: actions/checkout@v2

    - name: Lint changelog file
      uses: docker://avtodev/markdown-lint:v1 # fastest way
      with:
        rules: '/lint/rules/changelog.js'
        config: '/lint/config/changelog.yml'
        args: './CHANGELOG.md'
        ignore: './one_file.md ./another_file.md' # multiple files must be separated with single space

    # Or using current repository as action:

    - name: Lint changelog file
      uses: avtodev/markdown-lint@v1
      with:
        rules: '/lint/rules/changelog.js'
        config: '/lint/config/changelog.yml'
        args: './CHANGELOG.md'
```

Reply:
It will be a GitHub action, but we must be able to run it locally as well.

Reply:
Ah ok cool.